src/Http/Client/Response.php

```diff
@@ -115,7 +115,7 @@ class Response extends Message implements ResponseInterface
     /**
      * Cached decoded JSON data.
      *
-     * @var array
+     * @var array<mixed>
```

Comment:
This doesn't seem to provide any additional info over `array`.

Reply:
It does, as it states we already looked at this type and cannot find it more concrete
We still have a few hundred, maybe thousand of generic ones that wait to be checked and asserted
Once we did, there should be no more array itself actually, always defined
This way we can remove the

	checkMissingIterableValueType: false
	
config
which we should

Reply:
Practically no type is mixed type. So it's just needed to please phpstan, oh well.

Reply:
Well, we have a few cases where the type is actually mixed, and cannot be specified further.
That's why would also like to have `list<>` used where applicable, so that we know also the type here isnt further in need of being looked at.
Less work in the future to get to the fully documented level.